src/core/workspace/WorkspaceRootManager.ts

```diff
@@ -64,43 +64,6 @@ export class WorkspaceRootManager {
 		return VcsType.None
 	}
 
-	/**
-	 * Add a new workspace root
-	 */
-	async addRoot(rootPath: string, name?: string): Promise<void> {
```

Comment:
Got rid of this because for now supporting this along with checkpoints is too confusing.